aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -153,14 +182,22 @@ module aptos_framework::aptos_coin {
     use aptos_framework::aggregator_factory;
 
     #[test_only]
-    public fun initialize_for_test(aptos_framework: &signer): (BurnCapability<AptosCoin>, MintCapability<AptosCoin>) {
+    public fun initialize_for_test(aptos_framework: &signer)
+        : (
+        BurnCapability<AptosCoin>,
+        MintCapability<AptosCoin>
+    ) {
         aggregator_factory::initialize_aggregator_factory_for_test(aptos_framework);
         initialize(aptos_framework)
     }
 
     // This is particularly useful if the aggregator_factory is already initialized via another call path.
     #[test_only]
-    public fun initialize_for_test_without_aggregator_factory(aptos_framework: &signer): (BurnCapability<AptosCoin>, MintCapability<AptosCoin>) {
+    public fun initialize_for_test_without_aggregator_factory(aptos_framework: &signer)
+        : (
+        BurnCapability<AptosCoin>,
+        MintCapability<AptosCoin>
+    ) {
```

Comment:
Seems off, should be something like

```
public fun initialize_for_test_without_aggregator_factory(
    aptos_framework: &signer,
): (BurnCapability<AptosCoin>, MintCapability<AptosCoin>) {
```

Reply:
fixed at https://github.com/movebit/movefmt/commit/9ca658736a254a5696676f95c8da7b498a90281c